requirements/base.txt

```diff
@@ -1,5 +1,5 @@
 lxml==4.6.4
-psycopg2==2.9.1
+psycopg2-binary==2.9.1
```

Comment:
Hello, I'm just curious :) why this change ?

Reply:
It is because `psycopg2-binary` is installed as dependency in Dockerfile here https://github.com/IGNF/route-graph-generator/blob/develop/docker/debian/Dockerfile#L70

Do you know why there are `psycopg2` and `psycopg2-binary` used in container ?

Reply:
Oh ok ! I understand my misconception. base.txt is for a personn who want to use the project. I forgot that when I asked my question. Indeed, psycopg2 is used for connections to a base inside the r2gg scripts. ^^